examples/ccsd_T1_einsum_idx.py

```diff
@@ -0,0 +1,69 @@
+from fractions import Fraction
+from wick.expression import AExpression
+from wick.wick import apply_wick
+from wick.convenience import one_e, two_e, E1, E2, braE1, commute
+
+PYTHON_FILE_TAB = "    "
+
+
+def space_idx_formatter(name, space_list):
```

Comment:
I think this function can be done in one line, something like: `return name + str([space + "_idx" for space in space_list])`